pkg/distributor/distributor.go

```diff
@@ -2250,14 +2254,20 @@ func approximateFromZones[T ~float64 | ~uint64](isMultiZone bool, replicationFac
 				max = seriesCount
 			}
 		}
-		return max
+		if zonesTotal <= replicationFactor {
+			return max
+		}
+		// When the number of zones is larger than the replication factor, we factor
+		// that into the approximation. We multiply the max of all zones to a ratio
+		// of number of zones to the replication factor to approximate how series
+		// are spread across zones.
+		return T(math.Round(float64(max) * float64(zonesTotal) / float64(replicationFactor)))
```

Comment:
I think this change is fine (correct) for the case ingest storage is not in use, so I would keep it.